cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
The `max_length` for the `name` field in the `Genre` model should be 100 to match the migration file, which specifies a `max_length` of 100 for this field.